packages/astro/src/core/add/index.ts

```diff
@@ -637,13 +637,14 @@ export async function validateIntegrations(integrations: string[]): Promise<Inte
 				if (!parsed) {
 					throw new Error(`${bold(integration)} does not appear to be a valid package name!`);
 				}
-
 				let { scope, name, tag } = parsed;
-				let pkgJson = null;
-				let pkgType: 'first-party' | 'third-party' = 'first-party';
+				let pkgJson;
+				let pkgType: 'first-party' | 'third-party';
 
-				if (!scope) {
-					const firstPartyPkgCheck = await fetchPackageJson('astrojs', name, tag);
+				if (scope && scope !== '@astrojs') {
+					pkgType = 'third-party';
+				} else {
+					const firstPartyPkgCheck = await fetchPackageJson('@astrojs', name, tag);
```

Comment:
the `if (firstPartyPkgCheck instanceof Error) {` error handling below doesn't make sense any more. This branch is only hit if it _is_ an official Astro package.

Reply:
@FredKSchott Ah, this is to verify if it's _actually_ a published package under `@astrojs`! Without this check, we'd attempt to add `@astrojs/potato`, `@astrojs/fake`, etc.